README.md

```diff
@@ -172,11 +172,11 @@ To run this project, you'll need to set up some environment variables. These var
    npm run build:env
    ```
```

Comment:
According to latest `package.json`, this should be `build:env:dev`

Reply:
fixed